A.css

```diff
@@ -0,0 +1,7 @@
+h1 {
+  font-size: 3em;
+  font-family: Helvetica, sans-serif;
+  color: #5a09fe;
+  text-align: center;
+  margin-top: 25%;
+}
```

Comment:
- Good job styling your project, your code looks clean, please make sure you include one class for your header.